lib/diagnostic.js

```diff
@@ -17,7 +17,19 @@
 // Methods: sell, call
 
 // Answer Starts Here
-//
+// const calender =
+// {
+// 	namePerson :
+// 		dateEntered: "1234-56-78";
+// 		task: " ";
+// 		enddate: "8765-43-21";
+// 		calculateEndDate();
+// 	namePerson :
+// 		date: "1234-56-78";
+// 		task: "";
+// 		endDate: "8765-43-21";
+// 		calculateEndDate();
+// }
 // Answer Ends Here
```

Comment:
I think see where you were going with this, but all we were looking for is a bluerint like this

```

//
// EXAMPLE
// Description: Our app manages car sales.
// Entity: Car
// Attributes: make, model, year
// Methods: drive, brake, park
//
// Entity: SalesPerson
// Attributes: name, commissionRate
// Methods: sell, call
```